### PR TITLE
8286575: Document how properties in java.security are parsed

### PR DESCRIPTION
**Pull request by coffeys**
A doc edit to indicate that modifications of values in java.security conf file would require a JVM restart in order for such changes to be detected.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8286575](https://bugs.openjdk.org/browse/JDK-8286575): Document how properties in java.security are parsed

### Reviewers
 * [Sean Mullan](https://openjdk.org/census#mullan) (@seanjmullan - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11208/head:pull/11208` \
`$ git checkout pull/11208`

Update a local copy of the PR: \
`$ git checkout pull/11208` \
`$ git pull https://git.openjdk.org/jdk pull/11208/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11208`

View PR using the GUI difftool: \
`$ git pr show -t 11208`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11208.diff">https://git.openjdk.org/jdk/pull/11208.diff</a>

</details>

### CODE REVIEW

File: src/java.base/share/conf/security/java.security [hunk 1]
```diff
@@ -25,6 +25,8 @@
 #
 # If this properties file fails to load, the JDK implementation will throw
 # an unspecified error when initializing the java.security.Security class.
+# Properties in this file are typically parsed only once. Any modification
+# would require an application restart to reflect changes.
```
Comment (seanjmullan) [inline src/java.base/share/conf/security/java.security:29]: Maybe change the second sentence to "If any of the properties are modified, applications should be restarted to ensure the changes are properly reflected."

This makes it more consistent with the words in the first sentence.

Comment (coffeys) [reply src/java.base/share/conf/security/java.security]: Thanks for the review. Change implemented.

**Review by seanjmullan — APPROVED** [review]
Add a noreg-doc label to the issue. Looks good otherwise.

**Pull request closed without merging**